src/routes/docs/quick-starts/swift/+page.markdoc

```diff
@@ -0,0 +1,207 @@
+{% /only_dark %}
+{% only_light %}
+![Server integrations](/images/docs/quick-starts/integrate-server.png)
+{% /only_light %}
+| Category  {% width=120 %} | Required scopes       | Purpose |
+|-----------|-----------------------|---------|
+| Database  | `databases.write`     | Allows API key to create, update, and delete [databases](/docs/products/databases/databases). |
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to read [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+{% /section %}
+{% section #step-2 step=2 title="Create Swift project" %}
+Create a Swift CLI application by opening **XCode** > **Create a new XCode project**
+> **macOS** > **Command Line Tool**.
+
+Follow the wizard and open your new project.
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+
+Install the Swift Appwrite SDK by going to **File** > **Add Packages...** and search for the repo url 
+`https://github.com/appwrite/sdk-for-swift` and select `sdk-for-swift`.
+
+{% /section %}
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier. 
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Open the file `main.swift` and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```swift
+import Foundation
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1")
+    .setProject("<YOUR_PROJECT_ID>")
+    .setKey("<YOUR_API_KEY>")
+```
+
+{% /section %}
+{% section #step-5 step=5 title="Initialize database" %}
+
+Once the Appwrite Client is initialized, create a function to configure a todo collection.
+
+```swift
+let databases = Databases(client)
+
+var todoDatabase: Database?
+var todoCollection: Collection?
+
+todoDatabase = try? await databases.create(
+    databaseId: ID.unique(),
+    name: "TodosDB"
+)
+todoCollection = try? await databases.createCollection(
+    databaseId: todoDatabase!.id,
+    collectionId: ID.unique(),
+    name: "Todos"
+)
```

Comment:
Let's move these into `prepareDatabase` for consistency

Reply:
Done